modules/repository/generate.go

```diff
@@ -191,10 +195,24 @@ func generateRepoCommit(ctx context.Context, repo, templateRepo, generateRepo *r
 						}
 
 						if err := os.WriteFile(path,
-							[]byte(generateExpansion(string(content), templateRepo, generateRepo)),
+							[]byte(generateExpansion(string(content), templateRepo, generateRepo, false)),
```

Comment:
Why here use "false" for file naming escaping (actually sanitizing)?

It seems that the rule can be applied to all.